app/src/main/java/com/example/firstapplication/MainActivity.java

```diff
@@ -18,25 +18,28 @@ protected void onCreate(Bundle savedInstanceState) {
     }
```

Comment:
The purpose of this class has changed. Let's rename to `RegisterUserActivity`. Same applies to it's layout: `activity_register_user`

Reply:
okay will do that